### PR TITLE
country code selection dropdown for phone number added 

### LINKED ISSUE #262: Country code selection in user add and edit forms
**Description**
The App Panel has a section for managing application users. Currently the Add and Edit forms have hardcoded country code as "+91" which cannot be modified.

**Feature**
The country code should be a selectable field anb the desired country can be selected by searching its name. Pls refer to the reference interaction below:
ScreenRecording2024-06-14at3 52 20PM-ezgif com-video-to-gif-converter

### PR DESCRIPTION
**Pull request by deepakdinesh1123**
closes #262

### CODE REVIEW

File: backend/src/zango/core/utils.py [hunk 1]
```diff
@@ -121,3 +122,15 @@ def generate_lockout_response(request, credentials):
         {"logout_url": "/auth/logout", "cooloff_time": cooloff_time},
         status=403,
     )
+
+
+def validate_phone(phone_number):
```
Comment (shahharsh176) [inline backend/src/zango/core/utils.py:127]:
```python
def validate_phone(phone_number, region=None):
    try:
        region = region or settings.PHONENUMBER_DEFAULT_REGION
        phone_number = phonenumbers.parse(phone_number, region=region)
        if phonenumbers.is_valid_number(phone_number):
            return True
    except:
        return False
```

Also, can you add arguments info in docstring  ?

**Review by shahharsh176 — CHANGES_REQUESTED** [review]

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10447285227). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10632129530). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10632170174). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/assets/app_panel/js/build.1723136204750.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723136204750.min.js:1]: @adhiraj23zelthy Please remove the build

File: backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js:1]: Is there any changes done in this file?
if not then we can revert this

File: frontend/src/metadata.json [hunk 2]
```diff
@@ -1 +1 @@
-{ "buildMajor": 0, "buildMinor": 3, "buildPatch": 0, "buildTag": "" }
+{"buildMajor":0,"buildMinor":3,"buildPatch":1,"buildTag":""}
```
Comment (shahharsh176) [inline frontend/src/metadata.json:1]: please revert changes in this file

**Review by shahharsh176 — CHANGES_REQUESTED** [review]

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10682322935). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10683982167). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10699381300). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10819353602). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10849465233). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10903088750). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/11030074739). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/11031653026). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/11033447115). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/11033832260). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Review by shahharsh176 — APPROVED** [review]

**Pull request merged**